### ISSUE TITLE
New driver support (Nvidia Grid)

### ISSUE THREAD

**Issue by corbinwunderlich** [keylase/nvidia-patch#877]
**Please provide details about your environment (please complete the following information)**
Please add support for the nvidia grid guest drivers. They can be downloaded from https://cloud.google.com/compute/docs/gpus/grid-drivers-table

- Nvidia driver version: 550.90.07-grid (grid 17.3)
- GPU Model: RTX 2060 (spoofed to RTX 6000)
- OS: NixOS Unstable
- [x] I'm willing to participate in testing of new patch version

**Comment by Gelmo** [comment]
@corbinwunderlich Does the patch for the consumer 550.90.07 not work?

**Comment by corbinwunderlich** [comment]
> [@corbinwunderlich](https://github.com/corbinwunderlich) Does the patch for the consumer 550.90.07 not work?

GRID is a different driver. https://docs.nvidia.com/vgpu/index.html

**Comment by Gelmo** [comment]
> > [@corbinwunderlich](https://github.com/corbinwunderlich) Does the patch for the consumer 550.90.07 not work?
> 
> GRID is a different driver. [docs.nvidia.com/vgpu/index.html](https://docs.nvidia.com/vgpu/index.html)

Yes I understand, but the regex we have for the 550.90.07 consumer-grade driver works for the grid driver:
```
gelmo@desktop:~/tmp/nvidia/NVIDIA-Linux-x86_64-550.90.07-grid
❯ rafind2 -x "83fe01730848" libnvidia-fbc.so.550.90.07
0xe30b

gelmo@desktop:~/tmp/nvidia/NVIDIA-Linux-x86_64-550.90.07-grid
❯ rafind2 -x "e82554feff85c04189" libnvidia-encode.so.550.90.07
0x20776
```

I recommend trying to use the patch scripts on your driver and reporting back after testing.
